### ISSUE TITLE
[FEAT]: Expert Train & Inference for Sarc dataset with BLIP2

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#88]
### Description

Use Sarc training split of R, U, and S to train model ALBEF seperately and save to

Sarc R finetuned BLIP2
Sarc U finetuned BLIP2
Sarc S finetuned BLIP2

Then record the performance of the three models on the test set.
